<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcDeposit</class>
 <widget class="QWidget" name="CalcDeposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>572</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>572</width>
    <height>512</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DepositCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color:rgb(153, 193, 241);
  border: 1px solid gray;
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Сумма вклада:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Calculation_option">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
   font: 75 14pt &quot;Ubuntu Mono&quot;;
   background-color:rgb(255, 190, 111);
   border: 1px solid gray;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сумма налога</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сумма на вкладе к концу срока</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_time">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray;
   font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Вариант расчёта:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(143, 240, 164);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 11pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>RETURN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_answer">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Срок размещения:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_percent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray;
   font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_sum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray;
   font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(246, 97, 81);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 11pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_nal_percent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray;
   font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Налоговая ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Периодичность выплат:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 15pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 15pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Calculation_option_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
   font: 75 14pt &quot;Ubuntu Mono&quot;;
   background-color:rgb(255, 190, 111);
   border: 1px solid gray;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Ежемесячно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежеквартально</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежегодно</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 13pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>Выбрать пополнение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 13pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>Сделать частичное снятие</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>₽, руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Капитализация процентов:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 13pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_no">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 13pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_month">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	border: 0px
}</string>
   </property>
   <property name="text">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_year">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	border: 0px
}</string>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
